zipkin-core/src/main/scala/org/http4s/zipkin/core/interpreters/collector/Http.scala

```diff
@@ -0,0 +1,35 @@
+package org.http4s.zipkin.core.interpreters.collector
+
+import argonaut.Argonaut._
+import org.http4s.Uri.RegName
+import org.http4s.client.Client
+import org.http4s.zipkin.core.algebras.Collector
+import org.http4s.zipkin.core.interpreters._
+import org.http4s.zipkin.core.ZipkinInfo
+import org.http4s.{MediaType, Method, Request, Uri}
+
+import scalaz.concurrent.Task
+
+final case class Http(client: Client) extends Collector {
+
+  override def send(zipkinInfo: ZipkinInfo): Unit = {
+    val uri = Uri(
+      authority = Option(Uri.Authority(
+        host = RegName("localhost"),
+        port = Option(9411)
+      )),
+      path = "/api/v1/spans"
+    )
+    val payload = List(zipkinInfo).asJson.nospaces
```

Comment:
Does that `nospaces` make it a String?  I think the `asJson` is all you need here.